lambdatimes/src/components/Content/Tab.js

```diff
@@ -1,22 +1,44 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 
-const Tab = props => {
+ const Tab = props => {
+  if (this.props.tab === selectedTab) {
+    return className='tab active-tab';
+  } else {
+    return className='tab';
+  }
   /* Using your props, determine if the `tab` prop matches the `selectedTab` prop, 
       if they match, the className should be: 'tab active-tab', 
       if it is not it should just be 'tab'*/
   return (
     <div
-      className={''}
+      className={'tab'}
       onClick={() => {
-        /* Replace this dummy click handler function with your selectTabHandler function from props 
-         you'll need to pass the `tab` in as an argument to this handler. */
+        /* Replace this dummy click handler function with your 
+        selectTabHandler function from props 
+         you'll need to pass the `tab` in as an argument 
+         to this handler. */
+
+         this.props.selectedTabHandler(this.props.tab)
```

Comment:
Pay attention to the way you are accessing props, and consider what kind of component you are using.